modules/programs/delta.nix

```diff
@@ -135,14 +135,21 @@ in
       })
 
       (lib.mkIf (cfg.enable && cfg.enableJujutsuIntegration) {
+        assertions = [
+          {
+            assertion = config.programs.jujutsu.enable or false;
+            message = "programs.delta.enableJujutsuIntegration requires programs.jujutsu.enable to be true";
+          }
+        ];
```

Comment:
-1 for me here, I would like to always have `enableJujutsuIntegration`, and it being a no-op if JJ is disabled.